knowledge-base/grid-rows-text-ellipsis.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Prevent the Grid from wrapping text in multiline
+description: How to prevent the Grid from wrapping text in multiline
+type: how-to
+page_title: Prevent the Grid from wrapping text in multiline
+slug: grid-kb-rows-text-ellipsis
+position: 
+tags: 
+ticketid: 1499434
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Grid for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+I am using the TelerikGrid component for Blazor. One of my columns contains a long text, like a description and the Grid wraps it in multiple lines. I would like to prevent that from happening and show the maximum amount of text depending on the column width and after that put an ellipsis (...). 
+
+## Solution
+
+In order to prevent the Grid from wrapping the text in multiple lines you can use CSS and target the `<td>` HTML tags, which contain the data. In order to make that easier you can take advantage of the [OnCellRender event]({%slug grid-column-events%}#oncellrender) that the component exposes. The solution below showcases a sample implementation.
+
+
+````CSHTML
+@*Use the OnCellRender event to pass a custom CSS class to the Grid column and prevent it from wrapping the text in multiple lines for the Notes column*@
+
+<TelerikGrid Data="@MyData" Height="400px"
+             Pageable="true" Sortable="true" Groupable="true"
+             FilterMode="Telerik.Blazor.GridFilterMode.FilterRow"
+             Resizable="true" Reorderable="true">
+    <GridColumns>
+        <GridColumn Field="@(nameof(SampleData.Id))" Width="120px" />
+        <GridColumn Field="@(nameof(SampleData.Name))" Title="Employee Name" Groupable="false" />
+        <GridColumn Field="@(nameof(SampleData.Team))" Title="Team" />
+        <GridColumn Field="@(nameof(SampleData.Note))" Title="Note" OnCellRender="@OnCellRenderHandler" />
+        <GridColumn Field="@(nameof(SampleData.HireDate))" Title="Hire Date" />
+    </GridColumns>
+</TelerikGrid>
+
+@code {
+    void OnCellRenderHandler(GridCellRenderEventArgs args)
```

Comment:
consider moving this to a one-line lambda in the column declaration

Reply:
I used the "long event syntax" because I believe it is easier to understand by the clients. If it seems to be a bit too much we can do it in a lambda expression